packages/aws-cdk-lib/aws-ec2/lib/instance.ts

```diff
@@ -317,6 +317,19 @@ export interface InstanceProps {
    */
   readonly ebsOptimized?: boolean;
 
+  /**
+   * If you set this parameter to true , you can't terminate the instance using the Amazon EC2 console, CLI, or API;
+   *
+   * otherwise, you can. To change this attribute after launch, use [ModifyInstanceAttribute](https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_ModifyInstanceAttribute.html).
+   * Alternatively, if you set InstanceInitiatedShutdownBehavior to terminate, you can terminate the instance
+   * by running the shutdown command from the instance.
```

Comment:
Your wording and formatting seemed a bit off, but feel free to update my suggestion further. And just to be clear, is `instanceInitiatedShutdownBehavior` going to override this parameter? If that's the case, it should be made more explicit here.

```suggestion
   * If true, the instance will not be able to be terminated using the Amazon EC2 console, CLI, or API.
   *
   * To change this attribute after launch, use [ModifyInstanceAttribute](https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_ModifyInstanceAttribute.html).
   * Alternatively, if you set InstanceInitiatedShutdownBehavior to terminate, you can terminate the instance
   * by running the shutdown command from the instance.
```

Reply:
Modified comment